_travis/upload_coverage.sh

```diff
@@ -3,6 +3,6 @@
 set -exo pipefail
 
 if [[ -e .coverage ]]; then
-    python -m pip install codecov
-    codecov --env TRAVIS_OS_NAME,NOX_SESSION
+    python3.6 -m pip install codecov
+    python3.6 -m codecov --env TRAVIS_OS_NAME,NOX_SESSION
```

Comment:
This change is incorrect, we want to use "python" in order to also support the Python 2 builds

Reply:
Ohh, yeah I missed that